pages/app-layout/global-breadcrumbs.page.tsx

```diff
@@ -0,0 +1,45 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useState } from 'react';
+import AppLayout from '~components/app-layout';
+import SpaceBetween from '~components/space-between';
+import labels from './utils/labels';
+import BreadcrumbGroup from '~components/breadcrumb-group';
+
+export default function () {
+  const [extraBreadcrumb, setExtraBreadcrumb] = useState(false);
+  return (
```

Comment:
Maybe we could add a `<p>` to inform that this page only works when the `appLayoutWidget` flag is enabled (and maybe its current status?) - it could be confusing otherwise, making people think the feature is broken